README.md

```diff
@@ -1,3 +1,5 @@
+**Warning:** This package is soon-to-be renamed. Use [`@mdn/browser-compat-data`](https://www.npmjs.com/package/@mdn/browser-compat-data) instead. If you're already using `mdn-browser-compat-data`, read [the upgrade guide](https://github.com/mdn/browser-compat-data/blob/v1.1.0/UPGRADE-2.0.md).
```

Comment:
`soon-to-be` doesn't need hyphens in this context, imo. If it was in a context like "The soon-to-be-renamed browser-compat-data package", then it would have hyphens.

Reply:
Thank you, fixed with 638d3fcf017f50fd5cd370e78fe77963fcca3d1f. I wasn't sure if I was modifying _is_ or _renamed_.

Reply:
And fixed again with 2aba539b605e30950089e17c2d555bab9ac705ba.